doc/source/remote-instance.rst

```diff
@@ -0,0 +1,53 @@
+.. _pmm.amazon-rds-agentless:
+
+Adding Remote DB instance to |pmm|
+********************************************************************************
+
+There is a quick method for users to add DBaaS instances to |pmm| without having
+to hook into the Cloud Provider's API, and with no need to have |pmm-client|
+installed or any exporters running on the monitored node. The drawback of this
+approach is that you will not have visibility of host-level metrics (CPU,
+memory, and disk activity will not be captured nor displayed in |pmm|). 
+
+.. note: There is an alternative and more complex approach available for
+         |mysql-server|, which involves API-aware addition of an Amazon RDS /
+         Aurora DB instance.
+
+Both methods can be accessed in the |metrics-monitor| navigation menu by
+selecting the |pmm-add-instance| item in a |gui.pmm-dropdown| group:
+
+.. figure:: .res/graphics/png/metrics-monitor.cloud-menu.png
```

Comment:
update this image to be from the PMM menu

Reply:
replace image with this one:
![image](https://user-images.githubusercontent.com/4398871/47671091-6dd27b00-db74-11e8-8065-577086600004.png)
lets promote the two new screen updates by showing a highlight around them both, and around the pmm menu.